src/axom/mint/mesh/UnstructuredMesh.hpp

```diff
@@ -807,18 +838,20 @@ class UnstructuredMesh : public Mesh
    */
   virtual IndexType getNumberOfFaces() const final override
   {
-    return m_face_to_cell->getNumberOfIDs();
+    return m_faces.size();
   }
 
   /*!
    * \brief Return the capacity for faces.
    *
    * \note Codes must call initializeFaceConnectivity() before calling
    *       this method.
+   * \deprecated Has no significance, since adding external faces is not
+   *             supported. Use getNumberOfFaces() instead.
    */
-  virtual IndexType getFaceCapacity() const final override
+  [[deprecated]] virtual IndexType getFaceCapacity() const final override
```

Comment:
heh... my first encounter with a C++11 (and up) `[[attribute]]`.

Reply:
On this... can we start using `[[maybe_unused]]`?

Reply:
@kanye-quest C++ attribute supports [[deprecated("because...")]]. I think it would be better to include a brief statement about rationale and/or when it will be removed; e.g., next release.